cmd/vela-email/plugin.go

```diff
@@ -77,17 +79,17 @@ func (p *Plugin) Validate() error {
 	defer logrus.Trace("exited plugin.Validate")
 
 	logrus.Info("Validating Parameters...")
-	if len(p.Attachment.Filename) != 0 {
-		fileInfo, err := os.Stat(p.Attachment.Filename)
+	if len(p.EmailFilename) != 0 {
+		fileInfo, err := os.Stat(p.EmailFilename)
 		if errors.Is(err, os.ErrNotExist) {
```

Comment:
didn't see this until the next comment i made, but same question regarding other errors.

Reply:
not that I can think of, like above, any other error should get handled with email.NewEmailFromReader()